### ISSUE TITLE
Cannot add new user to Go Orchids

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/goorchids-app#145]
From Jay: "I am trying to add a new user for Go Orchids. It does not seem to work…I am in the staging admin area NOT the development site."

He says the same thing happens on the development site.

@chiruzzimarco Can you please troubleshoot and hopefully fix? We have ~2 hours left on the current contract, hoping that will be enough.

**Comment by chiruzzimarco** [comment]
@skleinfeldt The database is in an inconsistent state. I'm not sure how this happened, but i think it was due to the how the `plantshare` application database columns were originally created (related commits [#1](https://github.com/jazkarta/goorchids-app/commit/fb1e76717ba8bceeb0298e8fdbfc4188f44a24f7) and [#2](https://github.com/jazkarta/goorchids-app/commit/fb1e76717ba8bceeb0298e8fdbfc4188f44a24f7).

I fixed the problem by manually altering the database schema and now it is possible to create new users, but not to delete them.
Due to Python2.7 deprecation on Heroku i wasn't able to run a deployment and i suggest not spending any more time on the current environment, but going straight to the upgraded version (which is currently deployed on the dev environment) which will fix this and other issues as well as upgrading to Python3.
